Libraries/Utilities/createStrictShapeTypeChecker.js

```diff
@@ -47,9 +48,12 @@ function createStrictShapeTypeChecker(
     for (var key in allKeys) {
       var checker = shapeTypes[key];
       if (!checker) {
+        const stack =  parseErrorStack(new Error());
```

Comment:
cc @vjeux: drunk lint? Or is this just github being stupid and showing old irrelevant comments?